src/integration/pages/articlePage/user.amp.js

```diff
@@ -0,0 +1,9 @@
+const { amp } = global;
```

Comment:
This is a test file, but the filename makes it seem like its application code. Could it please be namespaced?
e.g. instead of `user.amp.js` `user.amp.test.js`?

Also, I'm not too clear of when something should be under the `user*.js` file and when it should be under the `service.test.js` file. Could you please clarify that?

Reply:
Sorry, thinking about it more, I realise that there's a reason that it doesn't have `.test.` - so it doesn't get run independently.

Reply:
It's actually a file that exports a function that when you call it runs the tests. If I added `.test` to the filename, jest would pick it up and try to run and then throw an error because nothing calls the function so I only added `.test` to the files that import these functions and call them. I can understand the confusion you have though. It is a pattern I've seen examples of elsewhere though

Reply:
Thanks. I'd like to know whether you plan for us to add further files other than `user.amp.js` and `user.canonical.js` e.g. `someothertypeoftest.amp.js`? I am not sure what specifically in these current files are scoped to `user`.

Reply:
The idea of the `user` naming was to indicate that these contain tests that are aimed at users e.g. testing we can see a headline or a hero image etc. I tried to separate the concerns of what we test instead of having everything mashed together for example testing SEO metadata, analytics, bundle scripts and other core application resources all in one file. It's just an idea though. I'm happy to take a different approach though or having a better name for the `user` file. I will of course add all this to the doc I'm writing.

Reply:
Okay, thanks for explaining. That all sounds great. I look forward to reading that doc.